src/logo/builtin.c

```diff
@@ -4505,6 +4505,15 @@ static const FFlogo S[] = {
             FF_COLOR_FG_WHITE,
         },
     },
+    // Shebang
+    {
+        .names = {"Shebang Linux", "Shebang", "shebang"},
```

Comment:
```suggestion
        .names = {"Shebang"},
```

`"Shebang"` is enough. Cases don't matter.